nano/styles/_icon.less

```diff
@@ -11,7 +11,7 @@
     background-image: url(uiLinkPendingIcon.gif);
 }
 
-.linkPending .uiIcon16 {
+.linkPending .fa {
```

Comment:
.. this'll actually make problems for me when trying to change the little spinner thing to fontawesome but whatever, deal with it as I come to it